cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -8,45 +8,46 @@ toc::[]
-+
-* A `--secure` flag is added for `odo url`. It indicates whether a URL is secure. 
-* `odo storage list` now displays the information about the storage state: `Pushed`, `NotPushed` or `Locally Deleted`.
-* `odo debug port-forward` now selects a port automatically if the default port (5858) is occupied.
-* The `--all` flag is renamed to  `--all-apps`.
-* The default PVC size has been increased to 10GiB.
-* The JSON output for `odo storage list -o json` has been restructured.
-* `nodejs-8` and `nodejs-10` images are no longer supported.
-* Experimental mode feature. By default, features under development or in experimental mode are hidden from the user. 
-+
-** To enable experimental mode, run:
-+
-----
-$ odo config set --env ODO_EXPERIMENTAL=true
-----
-+
-or
-+
-----
-$ odo preference set experimental true
-----
-** To disable experimental mode, run:
-+
-----
-$ odo config unset --env ODO_EXPERIMENTAL
-----
-+
-or
+* The `--devfile` flag is added to `odo create`. Run `odo create <component name> --devfile <devfile path>` to specify your devfile location. This flag is only available in the Experimental Mode. See, xref:odo-technology-preview_odo-release-notes[Technology Preview features] to learn how to enable it. 
+
+* Dynamic registry support. Now you can configure your own registries with the following commands:
 +
 ----
-$ odo preference set experimental false
+# Add registry to the registry list
+odo registry add <registry name> <registry URL>
+
+# List registry in the registry list
+odo registry list
+
+# Delete registry from the registry list
+odo registry delete <registry name>
+
+# Update registry in the registry list
+odo registry update <registry name> <registry URL>
+
+# List component with corresponding registry
+odo catalog list components
+
+# Create component that is hosted by specific registry
+odo create <component type> --registry <registry name>
 ----
 
-* `{odo-title}` now supports Ingress to create URLs on Kubernetes.
+* The `--starter` flag is added to `odo create`. Run `odo create nodejs --starter <project-name>` to download a source code of a project specified in the devfile. If no project name is specified, `odo` downloads the first one.
+
+* The `--context` flag is added to `odo push`. With `--context` you can trigger `odo push` from outside the source code directory. Run `odo push --devfile <path to the devfile> --context <directory with your component>` to specify the directory of your component. 
+
+* Performance improvement for `odo catalog list components` when using the devfiles.
+
+* The `--now` flag is added for `odo url delete` when using the devfiles.
+
+* `odo url delete --now` now works with the devfiles. 
+* The `--debug` flag now works with the devfiles.
+
+* Added machine readable output for listing Operator backed services. Run `odo catalog list services -o json` to display information about Operators and services in a JSON format.
```

Comment:
s/machine readable/machine-readable/
s/Operator backed/Operator-backed/
s/in a JSON format/in JSON format/